wagyu/cli/ethereum.rs

```diff
@@ -534,14 +546,14 @@ impl EthereumOptions {
     /// If `default` is enabled, then return the default path if no derivation was provided.
     fn to_derivation_path(&self, default: bool) -> Option<String> {
         match self.derivation.as_str() {
-            "ethereum" => Some(format!("m/44'/60'/0'/{}", self.index)),
+            "ethereum" => Some(format!("m/44'/60'/0'/0/{}", self.index)),
             "keepkey" => Some(format!("m/44'/60'/{}'/0", self.index)),
             "ledger-legacy" => Some(format!("m/44'/60'/0'/{}", self.index)),
             "ledger-live" => Some(format!("m/44'/60'/{}'/0/0", self.index)),
-            "trezor" => Some(format!("m/44'/60'/0'/{}", self.index)),
+            "trezor" => Some(format!("m/44'/60'/0'/0/{}", self.index)),
```

Comment:
The derivation path should be like this `m / 44' / coin_type' / account' / change / address`
https://wiki.trezor.io/Cryptocurrency_standards#BIP44_-_Multi-account_hierarchy_for_deterministic_wallets

Describing path levels:
https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels

Reply:
These are for bitcoin. While Ethereum uses the standard, it does its own derivation scheme

Reply:
No, there is only an open issue for Ethereum to transition to a new path:
https://github.com/ethereum/EIPs/issues/84#issue-143651804

Reply:
This seems to imply the default convention is to do without the `change` index. Can you revert the changes for now? As a custom path is already supported, those who need it can use it with `change`, and if the convention changes to support `change`, then this can be added back in

Reply:
Thanks for reverting